runtests.py

```diff
@@ -16,3 +17,9 @@
     else:
         args.insert(2, 'openwisp2')
     execute_from_command_line(args)
+    if not os.environ.get('SAMPLE_APP', False):
+        file_name = 'openwisp_controller/geo/tests'
+    else:
+        file_name = 'tests/openwisp2/sample_geo'
+    return_code = subprocess.call(f'pytest -c tests/pytest.ini {file_name}', shell=True)
```

Comment:
`sys.exit(pytest.main(['<path>']))` didn't work, I was getting database problems, database closed before all the queries executed, but this worked.